### PR TITLE
Ask Cocoa for terminated string contents

### PR DESCRIPTION
**Pull request by Catfish-Man**
Fixes rdar://74006308

### CODE REVIEW

**Comment by Catfish-Man** [comment]
@swift-ci please test

**Comment by Catfish-Man** [comment]
@swift-ci please benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please test

**Comment by Catfish-Man** [comment]
@swift-ci please benchmark

File: lib/Immediate/Immediate.cpp [hunk 1]
```diff
@@ -33,6 +33,7 @@
 #include "llvm/ExecutionEngine/Orc/DebugUtils.h"
 #include "llvm/ExecutionEngine/Orc/JITTargetMachineBuilder.h"
 #include "llvm/ExecutionEngine/Orc/LLJIT.h"
+#include "llvm/ExecutionEngine/Orc/ObjectTransformLayer.h"
```
Comment (milseman) [inline lib/Immediate/Immediate.cpp:36]: Why this addition?

**Review by milseman — APPROVED** [review]
LGTM, just not sure why that include is there.

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Pull request merged**